index.js

```diff
@@ -28,6 +28,7 @@ module.exports = {
         assert: 'either',
       },
     ],
+    'no-restricted-syntax': ['error', 'ForInStatement', 'LabeledStatement', 'WithStatement'],
```

Comment:
- `ForInStatement` can be covered by the `guard-for-in`
- `LabeledStatement` can be covered by the `no-labels` rule.
- `WithStatement` should be covered by the `no-with` rule

IMO, we should use those specific rules and save `no-restricted-syntax` for customizations not covered by specific rules.

Reply:
Sounds good. I'll see if we have those rules in place.

Reply:
Cool, it looks like we have all three of those in place already. Is the right thing to set this to `'no-restricted-syntax': 'off'`?

Reply:
I would say leave it how it was since I _think_ `eslint-config-airbnb` has some `no-restricted-syntax` rules. I _think_.

Reply:
Oh wait that's what you were overriding. Lol. Yeah off sounds good here.

Reply:
Right, I need to override the defaults, which is what I'm doing. I'll set to `off` then.

Reply:
I really wish we would define our own base config instead of relying on that of another company...oh well.

Reply:
Agreed. I don't really like airbnb's choices. They haven't aged well, to say the least.